services/github/src/pr-allowed.js

```diff
@@ -1,81 +0,0 @@
-const yaml = require('js-yaml');
-
-const DEFAULT_POLICY = 'collaborators';
-
-async function prAllowed(options) {
-  switch (await getRepoPolicy(options)) {
-  case 'collaborators':
-    return await isCollaborator(options);
-
-  case 'public':
-    return true;
-
-  default:
-    return false;
-  }
-}
-
-/**
- * Get the repo's "policy" on pull requests, by fetching .taskcluster.yml from the default
- * branch, parsing it, and looking at its `allowPullRequests`.
- */
-async function getRepoPolicy({login, organization, repository, instGithub, debug}) {
-  // first, get the repository's default branch
-  let repoInfo = (await instGithub.repos.get({owner: organization, repo: repository})).data;
-  let branch = repoInfo.default_branch;
-
-  // load .taskcluster.yml from that branch
-  let taskclusterYml;
-  try {
-    let content = await instGithub.repos.getContents({
-      owner: organization,
-      repo: repository,
-      path: '.taskcluster.yml',
-      ref: branch,
-    });
-    taskclusterYml = yaml.safeLoad(Buffer.from(content.data.content, 'base64').toString());
-  } catch (e) {
-    if (e.code === 404) {
-      return DEFAULT_POLICY;
```

Comment:
explanation: in this case it should return something like `null` because error 404 means there's no `.taskcluster.yml` in that branch.